src/components/shared/ContentItem/helper.js

```diff
@@ -25,13 +25,19 @@ export const getProgress = (unit, playTime) => {
   let progressIndicator = null;
 
   if (unit && playTime) {
-    localStorage.setItem(`${PLAYER_POSITION_STORAGE_KEY}_${unit.id}`, playTime);
     progressIndicator = (
       <Progress
         size="tiny"
         className="cu_item_progress"
         percent={playTime * 100 / unit.duration} />
     );
+
+    // if item had been fully played, start from the beginning
+    if (Math.abs(playTime - unit.duration) < 1) {
```

Comment:
@wissog you sure that unit have duration?

Reply:
it was used in <Progress the same way before my fix - percent={playTime * 100 / unit.duration} />
did something change, that we need to check duration